content/docs/studio/get-started.md

```diff
@@ -60,7 +60,7 @@ more projects._
 
    To connect to repositories on your self-hosted GitLab server, you must first
    add a connection to this server and create a team. Refer to the section on
-   [self-hosted GitLab server support](/doc/studio/user-guide/install-github-app)
+   [self-hosted GitLab server support](/doc/studio/user-guide/teams#manage-connections-to-self-hosted-gitlab-servers)
```

Comment:
Let's point directly to `/doc/studio/user-guide/connect-custom-gitlab-server`; the content in the link you used also points to that one.